src/app/pages/quickMaps/quickMaps-routing.module.ts

```diff
@@ -108,14 +108,14 @@ const routes: Routes = [
     ],
   },
   {
-    path: AppRoutes.INTERVENTION_REVIEW.getRouterPath(),
+    path: AppRoutes.QUICK_MAPS_COST_EFFECTIVENESS.getRouterPath() + '**',
     loadChildren: () =>
       import('./pages/costEffectiveness/interventionReview.module').then((m) => m.InterventionReviewModule),
   },
-  {
-    path: '**',
-    redirectTo: '',
-  },
+  // {
+  //   path: '**',
+  //   redirectTo: '',
+  // },
```

Comment:
If this wildcard catch is present the above routes for interventions don't seem to work :/

Probably needs fixing so as to not break the QuickMAPS default dropthrough